backend/server.js

```diff
@@ -39,6 +39,25 @@ app.get("/monthly-working-hours/:year/:years_back", (req, res) => {
   );
 });
 
+app.get("/billability-working-hours/:year/:years_back", (req, res) => {
+  let year = req.params.year;
+  const yearsBack = req.params.years_back;
+  let lastYear = Number(year) - Number(yearsBack);
```

Comment:
year and lastYear are a bit confusing variable names. Could you use startYear and endYear instead?